retroshare-gui/src/gui/common/RSTreeWidget.cpp

```diff
@@ -283,13 +283,57 @@ QMenu *RSTreeWidget::createStandardContextMenu(QMenu *contextMenu)
 		actShowHeader->setCheckable(true);
 		actShowHeader->setChecked(!isHeaderHidden());
 
-		QMenu *headerMenu = contextMenu->addMenu(QIcon(),tr("Show column..."));
-
 		QTreeWidgetItem *item = headerItem();
 		int columnCount = item->columnCount();
+

```

Comment:
Is this generic enough to work with all different instances of RSTreeWidget? It's used quite a lot.

Reply:
Yes